components/layout/variables.styl

```diff
@@ -98,7 +98,9 @@ privacy-light-color(prefix, attr)
   colors = 'closed' closed, 'public' public, 'private' private
   for value in colors
     {prefix + value[0]}
-      {attr} alpha(value[1], 0.1)
+      if attr == border-color
+        border 2px solid
+      {attr} alpha(value[1], 0.3)
```

Comment:
I never really understood what these helpers were trying to achieve. Wouldn't it be simpler to make a stylus function called `border-color`:

```stylus
borderColor(color)
  lighten(color, 30%)

// ...
.my-foo
  border-color borderColor(colors-channel-private)
```